src/fontcache.cpp

```diff
@@ -293,6 +293,10 @@ void FreeTypeFontCache::SetFontSize(FontSize fs, FT_Face face, int pixels)
 			pixels = Clamp(min(head->Lowest_Rec_PPEM, 20) + diff, scaled_height, MAX_FONT_SIZE);
 		}
 	}
+
+	/* Apply user-specified font zoom. */
+	pixels = pixels * (1 << _font_zoom);
```

Comment:
`*=` ?

Reply:
Or should it be one of (Un)ScaleByZoom here?